samples/display/ili9340/stm32f429i_disc1.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019 Pavlo Hamov <pavlo_hamov@jabil.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+&spi5 {
+	status = "okay";
+	cs-gpios = <&gpioc 2 0>;
+	ili9340@0 {
+		compatible = "ilitek,ili9340";
+		label = "DISPLAY";
+		spi-max-frequency = <10000000>;
```

Comment:
Why not put the max frequency supported by the ili display controller (15MHz [15151515])? Will this 10MHz not lead to a low baudrate selection on STM32 devices?

Reply:
Don't see any reason why not to. Thank you for noticing. Done